test/cli.js

```diff
@@ -30,262 +30,125 @@ describe('CLI', () => {
 
     it('runs a single test from the command line', (done) => {
 
-        const cli = ChildProcess.spawn('node', [labPath, 'test/cli/simple.js', '-m', '2000']);
-        let output = '';
-
-        cli.stdout.on('data', (data) => {
-
-            output += data;
-        });
-
-        cli.stderr.on('data', (data) => {
-
-            expect(data).to.not.exist();
-        });
-
-        cli.once('close', (code, signal) => {
-
-            expect(code).to.equal(0);
-            expect(signal).to.not.exist();
-            expect(output).to.contain('2 tests complete');
+        runCli(['test/cli/simple.js', '-m', '2000']).then((result) => {
+            expect(result.errorOutput).to.equal('');
+            expect(result.code).to.equal(0);
+            expect(result.output).to.contain('2 tests complete');
             done();
-        });
+        }).catch(done);
     });
 
     it('runs multiple tests from the command line', (done) => {
 
-        const cli = ChildProcess.spawn('node', [labPath, 'test/cli/simple.js', 'test/cli/simple2.js', '-m', '2000']);
-        let output = '';
-
-        cli.stdout.on('data', (data) => {
-
-            output += data;
-        });
-
-        cli.stderr.on('data', (data) => {
-
-            expect(data).to.not.exist();
-        });
-
-        cli.once('close', (code, signal) => {
-
-            expect(code).to.equal(0);
-            expect(signal).to.not.exist();
-            expect(output).to.contain('4 tests complete');
+        runCli(['test/cli/simple.js', 'test/cli/simple2.js', '-m', '2000']).then((result) => {
+            expect(result.errorOutput).to.equal('');
+            expect(result.code).to.equal(0);
+            expect(result.output).to.contain('4 tests complete');
             done();
-        });
+        }).catch(done);
     });
 
     it('runs a directory of tests from the command line', (done) => {
 
-        const cli = ChildProcess.spawn('node', [labPath, 'test/cli', '-m', '2000']);
-        let output = '';
-
-        cli.stdout.on('data', (data) => {
-
-            output += data;
-        });
-
-        cli.stderr.on('data', (data) => {
-
-            expect(data).to.not.exist();
-        });
+        runCli(['test/cli', '-m', '2000']).then((result) => {
 
-        cli.once('close', (code, signal) => {
-
-            expect(code).to.equal(0);
-            expect(signal).to.not.exist();
-            expect(output).to.contain('10 tests complete');
+            expect(result.errorOutput).to.equal('');
+            expect(result.code).to.equal(0);
+            expect(result.output).to.contain('9 tests complete');
```

Comment:
@Marsup Sharp 👀! [I had to delete the old "local only" test file](https://github.com/rluba/lab/commit/1dcd68ea7afe3bd29b9b200811bb26c8db071ba7) because it would have broken the "whole directory" test runs once it’s global.